contracts/LiquidityBridgeContract.sol

```diff
@@ -367,6 +368,32 @@ contract LiquidityBridgeContract {
         return transferredAmountOrErrorCode;
     }
 
+    function registerPegOut(
+        Quote memory quote,
+        bytes memory signature,
+        uint256 height
```

Comment:
the quote already has the depositHeightLimit (it should be a timestamp instead of a block number), you shouldn't pass the height as param.

Reply:
done